book/src/users/monitoring.md

```diff
@@ -234,10 +234,10 @@ This will serve metrics over port 3000.
 
 Generate a grafana dashboard. From trin root directory:
 ```sh
-cargo run -p trin-cli -- create-dashboard
+cargo run -p trin-cli -- create-dashboard http://localhost:3000 admin admin http://127.0.0.1:9090
```

Comment:
I know this is closed, but just a note: I don't think that last argument is right, it's going to have grafana looking for the prometheus server within its own container rather than going out to the docker host and looking there. That's how it works on my machine anyway, does it work differently on yours?

Reply:
@perama-v guide doesn't set it up with docker, it sets it as another user on the system.